crates/util/src/paths.rs

```diff
@@ -8,90 +8,6 @@ use serde::{Deserialize, Serialize};
 
 lazy_static::lazy_static! {
     pub static ref HOME: PathBuf = dirs::home_dir().expect("failed to determine home directory");
```

Comment:
The `HOME` path is left here, as it is needed by `PathExt`.

It's also not specific to Zed.